test/__tests__/uppy-server.js

```diff
@@ -0,0 +1,65 @@
+/*global jest:false, test:false, expect:false, describe:false*/
```

Comment:
> overly pollute the directory

I fear that this may be the case especially in a situation where a directory already contains a good number of files.

> spot which files are missing tests

`istanbul coverage` also does try to this for us so maybe we won't be missing out on much?